docs/templates/plugins.rst.j2

```diff
@@ -5,6 +5,10 @@
 {{ STEP | replace('-', ' ') | capitalize }} Plugins
 {{ '~' * (8 + (STEP | length)) }}
 
+{% if PARENT.__doc__ %}
```

Comment:
Maybe also temporarily skip rendering the text if it's equal to "Common parent of STEP.lower() plugins"? It has a nice intro for `provision`, the rest is just puzzling.

Reply:
Yeah, makes sense. Any hint about the proper expression? I tried in 9de48aa2 but I'm unable to make it work using the `jinja` syntax.